src/Hero6.UserInterface.SierraVga/ViewModel/TextBoxViewModel.cs

```diff
@@ -46,34 +47,32 @@ public string Text
 
         public void Show(string input)
         {
+
             Size size = FontManager.DefaultFont.MeasureString(input, 1, 1);
             int screenWidth = this.nativeWidth * (int)this.scale.X;
             int screenHeight = this.nativeHeight * (int)this.scale.Y;
-            int horizontalEmptySpace = screenWidth / 8;
-            int rowSize = screenWidth - (horizontalEmptySpace * 2);
-            int rowCount = (int)Math.Ceiling(size.Width / rowSize);
+            int screenColumnWidth = screenWidth / 8;
+            int displayAreaColumnWidth = screenWidth - (screenColumnWidth * 2);
+
+            var Padding = new Padding(1f);
```

Comment:
I've literally haven't used the "var" keyword a single place in the entire code base as we discussed and agreed a year ago that we preferred explicit typed variables. Are we really going to introduce this now?